ceremonies/src/lib.rs

```diff
@@ -509,10 +509,7 @@ pub mod pallet {
 			inactivity_timeout: InactivityTimeoutType,
 		) -> DispatchResultWithPostInfo {
 			<T as pallet::Config>::CeremonyMaster::ensure_origin(origin)?;
-			<InactivityTimeout<T>>::put(inactivity_timeout);
-			info!(target: LOG, "set inactivity timeout to {}", inactivity_timeout);
-			Self::deposit_event(Event::InactivityTimeoutUpdated(inactivity_timeout));
-			Ok(().into())
+			Self::do_set_inactivity_timeout(inactivity_timeout)
```

Comment:
I'm not a fan of scattering few lines of code into functions that are called just in one place. This was more readable before IMO

Reply:
Ah, didn't see that. Perfect argument